.travis.yml

```diff
@@ -7,7 +7,7 @@ dist: xenial
 cache: pip
 
 before_install:
-    - pip install docutils
+    - pip install docutils sphinx
```

Comment:
What about: 
```suggestion
    - pip install -r requirements.txt
```
in order to use the same versions of things? (modulo open ranges and caches, but close enough hopefully)

Reply:
Can we also use bionic (Ubuntu 18 LTS) instead of xenial (16 LTS)?

Happy to open another PR if deemed a significant enough change

Reply:
I suppose only if it’s required for this PR

Reply:
Not sure I'd say required, more just keeping the build image updated is good practice. I'll leave out for now